examples/dft/compile_time_dispatching/complex_fwd_buffer_mklgpu.cpp

```diff
@@ -29,7 +29,7 @@
 #include "oneapi/mkl.hpp"
 
 void run_example(const sycl::device& gpu_device) {
-    constexpr int N = 10;
+    constexpr std::size_t N = 10;
```

Comment:
Is there a reason why not using std::int64_t here and remove the cast below?

Reply:
`N` is also used to initialize 2 `std::vector` which requires an unsigned size. I think it's safer to use unsigned here so that we avoid using a negative size and try to allocate a huge size.